control-plane/drizzle/0020_talented_pandemic.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "jobs" ADD COLUMN "runtime_latency_ms" integer;
```

Comment:
I am curious of _"runtime"_ latency is the correct naming here?
Was also thinking _"handler"_ latency might make sense. 

<img width="194" alt="image" src="https://github.com/differentialhq/differential/assets/9162298/f12667ec-583a-4685-9183-43701c2e15fa">

Reply:
Latency probably isn't the correct term at all here.

Reply:
Renaming this to `function_execution_time`